source/iface/device_event.rst

```diff
@@ -2,23 +2,53 @@
   Copyright 2020 The Khronos Group Inc.
   SPDX-License-Identifier: CC-BY-4.0
 
-.. Moved to Synchronization and atomics
-
-.. rst-class:: api-class
-
-======================
+**********************
 ``sycl::device_event``
-======================
+**********************
 
 ::
 
-   class device_event;
+  namespace sycl {
+
+  class device_event {
+    device_event(__unspecified__);
+
+  public:
```

Comment:
I think indentation is missing on this line, at least.  Can you please update?

Reply:
Thanks! Fixed it. Should we also have a single space before ``public``/``private`` keywords?